src/__snapshots__/index.spec.js.snap

```diff
@@ -12,6 +12,18 @@ exports[`index named factory should instanciate a keyValue reducer (specified) 1
 
 exports[`index named factory should instanciate a simpleObject reducer 1`] = `"simpleObject"`;
 
+exports[`index suffix of simple factory should make a simple array init to empty 1`] = `Array []`;
+
+exports[`index suffix of simple factory should make a simple boolean init to false 1`] = `false`;
+
+exports[`index suffix of simple factory should make a simple object init to empty 1`] = `Object {}`;
+
+exports[`index suffix of simple factory should make a simple string init to blank 1`] = `""`;
+
+exports[`index suffix of simpleObject factory should make a simple object init to empty 1`] = `undefined`;
```

Comment:
This one should not be possible into redux

Reply:
I fix that